pyproject.toml

```diff
@@ -11,3 +11,9 @@ line-length = 160
 fast = true
 skip-string-normalization = true
 extend-exclude = "awx_collection"
+
+[tool.pip-tools]
```

Comment:
@oraNod looking closer, it seems like the repo maintainers chose not to shove everything into this file — there's a lot of tooling-specific files in the repo so I'd recommend moving this out.

Reply:
@webknjaz Sounds good. I wasn't really sure and thought maybe adding tooling-specific files for the docsite might get in the way or clutter the repo.